packages/app/src/components/Auth/AuthProvider.tsx

```diff
@@ -51,7 +51,7 @@ export const AuthProvider = ({ children }: AuthProviderProps) => {
         }
         const client = AuthClient.initialize(auth.clientId, auth.authority);
         client.onUserStateChanged.addListener((token?: AccessToken) => {
-          setIsAuthenticated(token?.isExpired(0) ?? false);
+          setIsAuthenticated(!!token);
           setAccessToken(token);
```

Comment:
if `isAutenticated === !!token`, do we need `isAuthenticated` at all ?

Seems like line `69` is no longer needed (was there for `isAttemptingSilentLogin` if I remember correctly) but, line `64` `signInSilent`, if succesful, will set the token value by `onUserStateChanged` anyway, and `signIn` will cause a redirect anyway...

Reply:
The only issue I saw in both PROD (with old Sign in button)  and QA (with current redirect) is that I ended up in a state where I was seemingly unauthenticated, but the redirect didnt proceed. I was wondering if it wasnt because the client was undefined so calling `AuthClient.signIn()`, didnt do anything because client was undefined... (Not sure there is anything clear to do here.

Reply:
I also questioned if we needed to keep `isAuthenticated` in the context, as well as `userInfo` because both of those can be derived from the `accessToken`. I would not be opposed to removing those, and deriving these from the accessToken

Will remove L69, because the onStateChanged listener should update accordingly, and no need for this fallback because of the manual sign in.

Not sure how to repro the issue you saw, only way i see the client to be undefined is if the auth config changes, which should never happen